docs/_milestone_.md

```diff
@@ -0,0 +1,85 @@
+# December Milestone
+
+## Overview
+
+- Complete the epics:
+  - `Add Support for Partitioned Database`
+  - `Inclusion of related models`
+  - `Production deployment`
+
+## Goals
+
+- [ ] Inclusion of related models [MVP]
+      https://github.com/strongloop/loopback-next/issues/1352
+
+  - [ ] [5]Reject create/update requests when data contains navigational
+        properties https://github.com/strongloop/loopback-next/issues/3439
+
+- [ ] Authentication & Authorization
+      https://github.com/strongloop/loopback-next/issues/3902
+
+  - [ ] [5]Contribute OpenAPI spec pieces from extensions
+        https://github.com/strongloop/loopback-next/issues/3854
+  - [ ] [3]Add user profile factory for authentication modules
+        https://github.com/strongloop/loopback-next/issues/3846
+
+- [ ] [Epic] Add Support for Partitioned Database
+      https://github.com/strongloop/loopback-connector-cloudant/issues/219
+
+  - [ ] [3]Optimize query by reading partition key from payload
+        https://github.com/strongloop/loopback-connector-cloudant/issues/217
+
+- [ ] Production deployment/logging/monitoring
+      https://github.com/strongloop/loopback-next/issues/1054
+
+  - [ ] [5][blog post] Show how to deploy a LB application in a cloud native
+        environment https://github.com/strongloop/loopback-next/issues/3715
+
+- [ ] From model definition to REST API with no custom repository/controller
+      classes https://github.com/strongloop/loopback-next/issues/2036
+
+  - [ ] [5]Model API booter & builder
+        https://github.com/strongloop/loopback-next/issues/3736
+
+- [ ] Migration guide between LB3 to LB4 [MVP]
+      https://github.com/strongloop/loopback-next/issues/453
+
+  - [ ] [3]Spike: Migration guide from LB3 - Authentication & authorization
+        https://github.com/strongloop/loopback-next/issues/3719
+  - [ ] [3]How to migrate user-defined model methods #3949
+
+- [ ] Miscellaneous
+  - [ ] [8]Spike: Discover and define models at runtime
+        https://github.com/strongloop/loopback-next/issues/2484
+  - [ ] [5]Column is always integer when running `npm run migrate`
+        https://github.com/strongloop/loopback-next/issues/2398
+
```

Comment:
Do we have any room for bug fixes and DX improvements? Here are my top picks.

**Bug fixes**

- Id deleted on update on MongoDB https://github.com/strongloop/loopback-next/issues/3267
- TodoList tutorial - GET ​/todos​/{id}​/todo-list gives unexpected response https://github.com/strongloop/loopback-next/issues/4147

**Developer experience**
- `[NEEDS ESTIMATE]` API Explorer ignores filter options where, fields, and order https://github.com/strongloop/loopback-next/issues/2208
_Continue the work started by the spike task in November to keep the ball rolling and don't loose hard-learned knowledge._
- `[NEEDS ESTIMATE]` Expose `isActive()` method in Transaction interface https://github.com/strongloop/loopback-next/issues/3471
- `[NEEDS ESTIMATE]` Missing `model.ts` after running `lb4 openapi` command https://github.com/strongloop/loopback-next/issues/4135
- `[NEEDS ESTIMATE]` Model strict: false is ignored with Postgre SQL repository https://github.com/strongloop/loopback-next/issues/4042
- `[NEEDS ESTIMATE]` Column names in lowercase using lb4 discover method - SQL connector https://github.com/strongloop/loopback-next/issues/3343